app/controllers/idv/welcome_controller.rb

```diff
@@ -7,6 +7,7 @@ class WelcomeController < ApplicationController
     include StepIndicatorConcern
 
     before_action :confirm_not_rate_limited
+    before_action :cancel_previous_in_person_enrollments
```

Comment:
What do you think about making this happen only on the show action?

```suggestion
    before_action :cancel_previous_in_person_enrollments, only: :show
```